action.yml

```diff
@@ -20,7 +20,7 @@ inputs:
     description: Used to pull node distributions from node-versions.  Since there's a default, this is typically not supplied by the user.
     default: ${{ github.token }}
   cache:
-    description: 'Used to specify a package manager for caching in the default directory. Supported values: npm, yarn'
+    description: 'Used to specify a package manager for caching in the default directory. Supported values: npm, pnpm, yarn'
```

Comment:
nit: since we're adding this one on, can we swap the order and add pnpm after yarn.  interesting question whether it's alphabetical or by usage.  not sure we can answer that but anecdotally, I think it should be npm, yarn, pnpm.  as I said a nit and not passionate about it.

Reply:
Thoughts on this as it relates to the README? I went with alphabetical order there as well.

Reply:
My 2 cents is just add to the list in those two instances and then 👍 on changes.  Thanks so much for this valuable addition.

Reply:
Perfect, I have updated the order in `action.yml` and the README to npm, yarn, pnpm 👍